deposit/src/test/java/edu/unc/lib/deposit/validate/VirusScanJobTest.java

```diff
@@ -205,6 +207,38 @@ public void failOneScanTest() throws Exception {
         }
     }
 
+    @Test
+    public void failAndRescanTest() throws Exception {
+        when(scanResult.getStatus()).thenReturn(Status.FAILED)
+                .thenReturn(Status.PASSED);
+        File textFile = new File(depositDir, "text.txt");
+        doReturn(scanResult).when(clamScan).scan(argThat(new FileArgumentMatcher(textFile)));
+
+        Model model = job.getWritableModel();
+        Bag depBag = model.createBag(depositPid.getRepositoryPath());
+
+        PID filePid = addFileObject(depBag, textFile);
+
+        job.closeModel();
+
+        try {
+            job.run();
+            fail();
+        } catch(JobFailedException e) {
+            verify(jobStatusFactory).setTotalCompletion(eq(jobUUID), eq(1));
+            verify(jobStatusFactory, times(0)).incrCompletion(eq(jobUUID), eq(0));
+        } finally {
```

Comment:
I don't think you want this in a finally block, it means that it will still run even if the `fail()` up above was called